besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -401,12 +401,18 @@ private void detectJemalloc(final List<String> lines) {
         .ifPresentOrElse(
             t -> {
               try {
-                final String version = PlatformDetector.getJemalloc();
-                lines.add("jemalloc: " + version);
+                String jemallocEnabled = environment.get("BESU_USING_JEMALLOC");
+                if (jemallocEnabled.equals("true")) {
+                  final String version = PlatformDetector.getJemalloc();
+                  lines.add("jemalloc: " + version);
+                } else {
+                  logger.info(
+                      "BESU_USING_JEMALLOC is present but is not set to true, jemalloc library not loaded");
```

Comment:
```suggestion
                      "BESU_USING_JEMALLOC={} , jemalloc library not loaded", jemallocEnabled);
```

Reply:
Well, the original message was

> "BESU_USING_JEMALLOC is present but we failed to load jemalloc library to get the version"

so whenever I change existing code I tend to keep the "style" coherent with what is there, including the comments and log messages, so I guess 

> "BESU_USING_JEMALLOC is present but is not set to true, jemalloc library not loaded"

is more close to the original one...